driver_cpl/cime_config/config_component.xml

```diff
@@ -409,7 +409,7 @@
   <entry id="DOUT_S"> 
     <type>logical</type>
     <valid_values>TRUE,FALSE</valid_values>
-    <default_value>TRUE</default_value>
+    <default_value>FALSE</default_value>
```

Comment:
This is turned off in the csh test. As an example - look at the ERI_script
and ERR_script you'll see calls like

./xmlchange -file env_run.xml -id DOUT_S        -val FALSE

On Thu, Jan 14, 2016 at 3:55 PM, James Foucar <notifications@github.com>
wrote:

> In driver_cpl/cime_config/config_component.xml
> <https://github.com/ESMCI/cime/pull/33#discussion_r49798543>:
>
> > @@ -409,7 +409,7 @@
> >    <entry id="DOUT_S">
> >      <type>logical</type>
> >      <valid_values>TRUE,FALSE</valid_values>
> > -    <default_value>TRUE</default_value>
> > +    <default_value>FALSE</default_value>
>
> How does your testing system turn it off?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ESMCI/cime/pull/33/files#r49798543>.
>